Makefile

```diff
@@ -96,6 +96,19 @@ deploy-dev: manifests
 	${KUSTOMIZE} build config/runtimes | kubectl apply -f -
 	if [ ${KSERVE_ENABLE_SELF_SIGNED_CA} != false ]; then ./hack/self-signed-ca.sh; fi;
 
+deploy-dev-raw: manifests
+	./hack/image_patch_dev.sh development-raw
+	# Remove the certmanager certificate if KSERVE_ENABLE_SELF_SIGNED_CA is not false
+	cd config/default && if [ ${KSERVE_ENABLE_SELF_SIGNED_CA} != false ]; then \
+	${KUSTOMIZE} edit remove resource certmanager/certificate.yaml; \
+	else ${KUSTOMIZE} edit add resource certmanager/certificate.yaml; fi;
+	${KUSTOMIZE} build config/overlays/development-raw | kubectl apply -f -
+	# TODO: Add runtimes as part of default deployment
+	kubectl wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
+	sleep 10 # For now 2 secs does not workout as webhook takes time to come up
+	${KUSTOMIZE} build config/runtimes | kubectl apply -f -
+	if [ ${KSERVE_ENABLE_SELF_SIGNED_CA} != false ]; then ./hack/self-signed-ca.sh; fi;
+
```

Comment:
I think it is probably better to reuse `make deploy-dev` and add a kubectl patch command to change the default config

Reply:
Yeah. Let me try that way.